### PR TITLE
Added new features to specify attachment files.

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
- Added `AttachmentFile` context as well as `File` context as output.
- Supported to search entire entries in the war room.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
@spearmin10 thank you for your contribution.
This is a complete rewrite of the script.
please note that since this is a script in a XSOAR supported pack, in order to accept changes in it, we need to make sure we preserve backward compatibility.
In this case it will require more unit testing and doing a demo of the usability.
Please see my preliminary comments and let's setup a demo call.

**Comment by ShahafBenYakir** [comment]
Hi @spearmin10 Do you need any help understanding the comments you have received from the reviewer? let us know what is holding you back.

**Comment by spearmin10** [comment]
I got an error with `Packs/CommonScripts/Scripts/RepopulateFiles/RepopulateFiles.yml: [BC100] - Possible backwards compatibility break, You've changed the subtype, please undo.` when changing `subtype: python2' to `subtype: python3', and
I wouldn't be able to update this without removing the test code because the architecture has changed. so I've created a new PR as a different script. Refer to: https://github.com/demisto/content/pull/15137

File: Packs/CommonScripts/Scripts/RepopulateFiles/RepopulateFiles.yml [hunk 1]
```diff
@@ -1,18 +1,77 @@
...
+  description: File name
+- contextPath: AttachmentFile.Attachment.path
+  description: File path
+- contextPath: AttachmentFile.Attachment.showMediaFile
+  description: showMediaFile
+- contextPath: AttachmentFile.Attachment.type
+  description: File content type
 scripttarget: 0
-type: python
 subtype: python2
```
[the remark this answers is not in the document]
Comment (spearmin10) [reply Packs/CommonScripts/Scripts/RepopulateFiles/RepopulateFiles.yml]: Changed to `python3`

**Comment by spearmin10** [comment]
@iyeshaya I've updated the PR following to your comment below.
I'm wondering if you review and approve this.
> I have consulted with the managers about this enhancement
> We have decided not to add a new script with similar features and instead enhance the existing script
> Can you open a new PR enhancing the exiting script, please ignore the error with subtype this would be fixed by the manager

**Review by iyeshaya — CHANGES_REQUESTED** [review]
Please fix the issue in https://app.circleci.com/pipelines/github/demisto/content/121033/workflows/b7974481-bc7f-4e9b-958c-d44c12c25793/jobs/435478

**Comment by iyeshaya** [comment]
Talked with @spearmin10,
We agreed to open a new PR with some changes to the original code.

**Pull request closed without merging**